tests/integration/telemetry/dashboard_test.go

```diff
@@ -339,15 +339,15 @@ func setupDashboardTest(t framework.TestContext) {
 			}).
 			BuildOrFail(t)
 	}
-	for _, ing := range ingr {
-		// Send 200 http requests, 20 tcp requests across goroutines, generating a variety of error codes.
-		// Spread out over 5s so rate() queries will behave correctly
-		g, _ := errgroup.WithContext(context.Background())
-		tcpAddr := ing.TCPAddress()
-		ticker := time.NewTicker(time.Second * 5)
-		for t := 0; t < 20; t++ {
-			<-ticker.C
-			g.Go(func() error {
+	// Send 200 http requests, 20 tcp requests across goroutines, generating a variety of error codes.
+	// Spread out over 20s so rate() queries will behave correctly
+	g, _ := errgroup.WithContext(context.Background())
+	ticker := time.NewTicker(time.Second)
```

Comment:
@howardjohn your ticker here should be times 20? according to your above comment,  `Spread out over 20s`

Reply:
we have a loop above this. So 20 iterations, every 1s

Reply:
got it! approval!